### ISSUE TITLE
summary is not a generic

### ISSUE THREAD

**Issue by turgeonmaxime** [r-lib/roxygen2#1197]
TLDR: Is there a tag that explicitly adds `S3method` when `roxygen2` doesn't recognize an S3 generic?

I am a developer for the `casebase` package, and I'm trying to create a `summary` method for our custom object (i.e. of class `singleEventCB`). When I add the tag `@export` in my documentation for the function `summary.singleEventCB`, the directive `export(summary.singleEventCB)` is added to the `NAMESPACE` instead of `S3method(summary,singleEventCB)`. Note that when `casebase` is loaded, `roxygen2` doesn't recognize `summary.singleEventCB` as an S3 method:
``` r
library(casebase)
#> See example usage at http://sahirbhatnagar.com/casebase/
roxygen2::is_s3_method("summary.singleEventCB")
#> [1] FALSE
```

<sup>Created on 2021-02-03 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

I think it's because we're using the `VGAM` package for fitting multinomial regression, and we added `importMethodsFrom(VGAM,summary)` in our `NAMESPACE`. 

In any case, if I manually update the `NAMESPACE` and add `S3method(summary,singleEventCB)`, everything works as expected. Is there a way to force `roxygen2` to specifically add this namespace directive, instead of `export(summary.singleEventCB)`?

**Comment by gaborcsardi** [comment]
I think you can use `@method`, see here: https://roxygen2.r-lib.org/articles/rd.html#s3

**Comment by turgeonmaxime** [comment]
Thanks for the suggestion, I think it worked! I added both `@method summary singleEventCB` and `@export` to my code, and I do get `S3method(summary,singleEventCB)`. 

Is this the intended use, adding both tags? It wasn't clear from the documentation, I sort of stumbled upon it by mistake (I forgot to remove the `@export` tag).
